### PR TITLE
Update triage dashboard python code to support Python 3.

### PR DESCRIPTION
**Pull request by dieseldesai**

### CODE REVIEW

**Comment by dieseldesai** [comment]
Completed the instructions for signing the CLA

File: triage/berghelroach.py [hunk 1]
```diff
@@ -301,7 +301,7 @@ def getDistance(self, target, limit):
             tmp = self.priorLeft
             self.priorLeft = self.lastLeft
             self.lastLeft = self.currentLeft
-            self.currentLeft = self.priorLeft
+            self.currentLeft = tmp
```
Comment (Katharine) [inline triage/berghelroach.py]: …huh. I wonder what impact this bug had, if any.

**Comment by dieseldesai** [comment]
Let me know if you'd rather remove the byteify parts of the code. I'm not sure what effect it might have on other code that uses this library, so I took the safer route of just changing the tests.

**Comment by dieseldesai** [comment]
Actually, just realized that the byteify code is only in the test. I'll update this PR to remove that. I agree that it doesn't seem necessary anymore.

**Comment by spiffxp** [comment]
I feel like the `/usr/bin/env python2` at the top of these files no longer makes sense, unless this is intended to allow triage to work with both?

**Comment by Katharine** [comment]
This should make it work with both.

**Comment by dieseldesai** [comment]
Yea, I tested manually with both Python 3 and Python 2. I'm guessing the presubmit tests are still using Python 2 because of https://github.com/bazelbuild/bazel/issues/4815

**Comment by spiffxp** [comment]
/lgtm
/approve
Will keep an eye on https://testgrid.k8s.io/sig-testing-misc#triage

**Pull request merged**